### PR TITLE
Task/add sqlalchemy/296

### PR DESCRIPTION
**Pull request by andrea-williams**

### CODE REVIEW

File: schemas.py [hunk 1]
```diff
@@ -107,7 +112,7 @@ class WeatherForecastValues(BaseModel):
     wind_speed_925mb: float = None
     wind_direction_925mb: float = None
     wind_speed_850mb: float = None
-    wind_direction_850m: float = None
```
Comment (Sybrand) [inline schemas.py:110]: You need to fix the mapping in forecast.py - line 60

File: tests/db/test_crud.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+          ))
+def test_create_single_forecast():
+    # NOTE: Once we do CRUD through the API, relook this test.
+    """ BDD Scenario. """
+
+@given("I am a session creating a forecast with <date_time>, <temperature>, <rh>, <wind_speed>, <total_precip> at time <issue_date> for station <station_code>")
+def create_single_forecast(date_time: str, temperature: float, rh: float, wind_speed: float,
+                           total_precip: float, issue_date: str, station_code: int):
+    """ Create a single forecast & mock adding it to the DB """
+    session = UnifiedAlchemyMagicMock()  # TODO: How far do we take this?
```
Comment (Kyubinhan) [inline tests/db/test_crud.py:28]: Does this UnifiedAlchemyMagicMock require any clean up after each transaction?

File: tests/conftest.py [hunk 3]
```diff
@@ -24,6 +24,11 @@ def mock_env(monkeypatch):
     monkeypatch.setenv("WFWX_BASE_URL", "http://localhost/wfwx")
     monkeypatch.setenv("WFWX_MAX_PAGE_SIZE", "1000")
     monkeypatch.setenv("KEYCLOAK_PUBLIC_KEY", "public_key")
+    monkeypatch.setenv("POSTGRES_HOST", "db")
```
Comment (Sybrand) [inline tests/conftest.py:27]: You can actually remove this, since it won't be applied.

File: tests/db/test_crud.py (hunk 2 above)
Comment (andrea-williams) [reply tests/db/test_crud.py:28]: nope! Because it's only mocking transactions with the database, not actually making them, there's no cleanup required :)

**Pull request merged**